src/web/Html.tsx

```diff
@@ -89,6 +92,43 @@ export const Html = React.forwardRef(
     const old = React.useRef([0, 0])
     const target = portal?.current ?? gl.domElement?.parentNode
 
+    const [opacity, setOpacity] = React.useState(1) 
+    
+    useFrame(() => {
+      
+      if (occluder?.current && group.current) {
+        v1.setFromMatrixPosition(group.current.matrixWorld)
+        v2.setFromMatrixPosition(camera.matrixWorld)
+
+        const dir = new THREE.Vector3()
+        
+        const ray = new THREE.Raycaster(v1, dir.subVectors(v2, v1).normalize())
+
+        const hit = ray.intersectObject(occluder.current)[0]
+
+        const x = new THREE.Object3D()
+        
+        if (typeof hit !== "undefined") {
+          x.position.copy(hit.point)
+          const position = calculatePosition(x, camera, size)
+          const targetPosition = calculatePosition(group.current, camera, size)
+
+          const v1 = new THREE.Vector2(...position)
+          const v2 = new THREE.Vector2(...targetPosition)
+          
+          setOpacity(v1.distanceTo(v2)  / 100)
+        } else {
+          setOpacity(1)
+        }
+        
+        // const dist = v1.distanceTo(p.point)
```

Comment:
Remove this in favore of a simple on/off

Reply:
"in favore" 🤌